bundles/org.openhab.binding.ecovacs/README.md

```diff
@@ -16,6 +16,8 @@ At this point, the following devices are fully supported and verified to be work
 - Deebot OZMO 950
 - Deebot OZMO Slim 10/11
 - Deebot N8 series
+- Deebot X2 Omni
```

Comment:
The PR doesn't add X2 to the device list, and it doesn't work properly either from my testing with a user (there's issues with downloading map images). I think it should be removed from the list.

Reply:
I saw that you added X2 and noticed that it was missing from the readme. Since you asked, I'll remove it.

Reply:
X2 removed